frontend/src/pages/Home.jsx

```diff
@@ -1,3 +1,25 @@
+import { useRef } from "react";
+
 export default function Home() {
-  return <p>Hello World</p>;
+  const searchValue = useRef();
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.warn(searchValue.current.value);
```

Comment:
I think that line shouldn't be there anymore.

Reply:
As said in previous comment, this line of code is acting as a placeholder for future integration